### ISSUE TITLE
Couldn’t open console and the file seems fail to update 

### ISSUE THREAD

**Issue by Christina221A** [Norbyte/bg3se#99]
I have put the DLL file in the folder. I confirmed its location. I tried to download from the latest release and tried again on bg3mm, both failed to work. I haven’t seen any pop-up windows in or out of the game. The only thing I can think of is that I don’t use the Steam Cloud saves because my saves are far too large to store online. Is there any connections between this and my issue?
Would it work if I just download the raw codes from GitHub and run it on my computer? Or is there another way to confirm the file has been updated or not?

**Comment by BelegCufea** [comment]
@Christina221A 
To enable console you need a file named `ScriptExtenderSettings.json` in the same folder as `DWrite.dll` (namely `..\Baldurs Gate 3\bin`). The content should include
```json
{
  "CreateConsole": true
}
```
The file is part of the `BG3SE-Updater-with-Console-v3.zip` archive you can dowload [here](https://github.com/Norbyte/bg3se/releases).

The console only displays debug messages. It can't be used to issue any commands.

As for any update/download issue, the `DWrite.dll` file is only loader of Script Extender. It will check, if you have current version and in case there is new one available, it will download and install new one into `%AppData%/Local/BG3ScriptExtender` folder.
If for any reason it can't reach or downolad new verison, you will see prompt when runnign BG3 (in the game).

### At your own risk (and I mean it)
If you have a problem with downloading Script Extender you can use offline verision of Script Extender available at [https://bg3se-updates.norbyte.dev/Channels/Release/ManualInstall.zip](https://bg3se-updates.norbyte.dev/Channels/Release/ManualInstall.zip).
This version includes `DWrite.dll` with current _(at the time of download)_ Script Extender already embedded and `libprotobuf-lite.dll` support library. Both of these has to be copied into `..\Baldurs Gate 3\bin`. Be aware, you still need `ScriptExtenderSettings.json` file in the same directory to see debug console.

**This offline version won't ever update!**
You will need to download and copy the files manualy when new version of Script Extender is available. And as it will not check for new versions, it is up to you if you will use this not supported method. All versions of Script extender are listed in [https://bg3se-updates.norbyte.dev/Channels/Release/Manifest.json](https://bg3se-updates.norbyte.dev/Channels/Release/Manifest.json) file. Newest is the latest.

**Comment by Norbyte** [comment]
> The console only displays debug messages. It can't be used to issue any commands.
This is not true, the console can be used to execute code in the extenders Lua context.

**Comment by BelegCufea** [comment]
@Norbyte 
Ah, sorry for disinformation. I am just plain BFU ...

Just a question. Should I put such a "instructions" as above here, or is it more harm then good situation?
